docs/index.md

```diff
@@ -11,6 +11,37 @@ UP4W can be installed from the [Microsoft Store](https://apps.microsoft.com/home
 
 WSL is preferred by many organisations as a solution to run a fully-functioning Linux environment on a Windows machine. UP4W empowers system administrators and corporate security teams to manage these WSL instances at scale.
```

Comment:
It was brought to my attention an issue in the following sentence from the previous paragraph:

> When the Landscape client is installed on the host, any connected WSL instances will be auto-enrolled in Landscape.

It suggests that there is some sort of additional Landscape client we install on Windows, which is not quite the case. UP4W is such a client and the statement should clarify that once we configure UP4W with Landscape information then any connected WSL instance will be auto-enroleld in Landscape. I'd rephrase it like:

> When UP4W is configured with the necessary information about Landscape, any connected WSL instances will be auto-enrolled in Landscape.

As it's unrelated to the changes fo this PR, feel free to bring it into a new PR or augment this one.

Reply:
I didn't comment in the proper line because it's outside of the context GitHub UI allows commenting on.

Reply:
That's great Carlos -- thanks. There is some important content revision work that needs to be done in line with the testing work. I will note this suggestion and incorporate it into a later PR.